dot/rpc/websocket.go

```diff
@@ -233,3 +240,36 @@ func (h *HTTPServer) blockReceivedListener() {
 		}
 	}
 }
+
+func (h *HTTPServer) storageChangeListener() {
+	if h.serverConfig.StorageAPI == nil {
+		return
+	}
+
+	for change := range h.storageChan {
+
+		if change != nil {
+			for i, sub := range h.serverConfig.WSSubscriptions {
```

Comment:
I'm wondering if it would be simpler to start a goroutine for each subscription, what do you think?

Reply:
Yes, I've refactor websockets to better handle this.